app/v1/connectors/ListBenefitsConnector.scala

```diff
@@ -40,9 +40,9 @@ class ListBenefitsConnector @Inject() (val http: HttpClient, val appConfig: AppC
     val nino    = request.nino.nino
     val taxYear = request.taxYear
 
-    getWithQueryParams(
+    get(
       IfsUri[ListBenefitsResponse[HMRCStateBenefit, CustomerStateBenefit]](s"income-tax/income/state-benefits/$nino/$taxYear"),
-      request.benefitId.map("benefitId" -> _).toSeq
+      queryParams = request.benefitId.map("benefitId" -> _).toSeq
```

Comment:
Ah, I am guessing QPs are set to 'Nil' in the BaseDownstreamConnector for the case that we don't provide a benefitId, right? However benefitId is not a QP so does it make this code confusing to read?

Reply:
If you call to Seq on an empty Option (i.e. None) you'll get an empty Seq, basically Nil. benefitId is a query parameter, downstream.